domain/machine/service/service.go

```diff
@@ -243,12 +243,12 @@ func (s *Service) AllMachineNames(ctx context.Context) ([]coremachine.Name, erro
 	return machines, nil
 }
 
-// InstanceId returns the cloud specific instance id for this machine.
+// InstanceID returns the cloud specific instance id for this machine.
 // If the machine is not provisioned, it returns a NotProvisionedError.
```

Comment:
**todo:** Qualify the godoc link [machineerrors.NotProvisionedError]

Reply:
Fixed in 9b4a2b06aa357bd3119ef601318c0f923c583ff7.